src/main/webapp/style.css

```diff
@@ -0,0 +1,103 @@
+#help-near-me-content {
+  font-family: 'Courier New', Courier, monospace;
+  color: #666666;
+  display: flex;
+  flex-direction: column;
+}
+ 
+#help-near-me-content a {
+  color: #666666;
+  text-decoration: none;
+  border-bottom: 3px solid transparent;
+}
+
+#help-near-me-content a:hover {
+  border-bottom: 3px solid #666666;
+}
+ 
+#nav-bar,
+#site-title {
+  display: flex;
+  justify-content: space-between;
+  font-size: 36px;
+  margin: 0 auto;
+  width: 650px;
+}
+
+#site-title {
+  justify-content: center;
+  margin-bottom: 10px;
+}
+
+#zipcode-form {
+  display: flex;
+  justify-content: center;
+  margin: 40px 50px 20px 50px;
+}
+
+#zipcode-entry {
+  width: 20%;
+  padding: 8px 10px;
+  margin: 8px 0;
+  display: inline-block;
+  border: 1px solid #ccc;
+  border-radius: 4px;
+  box-sizing: border-box;
+}
+
+#zipcode-label {
+  margin: auto 2px auto 0;
+  font-size: 24px;
+}
+
+#submit-button {
+  border: 1px solid #ccc;
+  background-color: #A9A9A9;
+  border-radius: 4px;
+  font-family:'Courier New', Courier, monospace;
+  color: #ffffff;
+  font-size: 16px;
+  cursor: pointer;
+  margin-left: 10px;
+  padding: 5px 20px;
+}
+
+#filter-button-area {
+  width: 80%;
+  margin-bottom: 10px;
+}
+
+#filter-button {
+  border: 1px solid #ccc;
+  background-color: #A9A9A9;
+  border-radius: 4px;
+  font-family:'Courier New', Courier, monospace;
+  color: #ffffff;
+  font-size: 16px;
+  cursor: pointer;
```

Comment:
this stuff seems like it repeats a lot of #submit-button, maybe it would be easier to add more buttons in the future if we used a class here?